docs/rst/tutorials/core/deployment/easy_mode/easy_mode.rst

```diff
@@ -0,0 +1,87 @@
+.. include:: ../../../../exports/alias.include
+
+.. _easy_mode_tutorial:
+
+Easy Mode Tutorial
+==================
+
+This tutorial aims to demonstrate the new :ref:`easy_mode` feature in *Vulcanexus* by running a simple talker-listener example demo.
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+Prerequisites
+-------------
+
+* It is recommended to have a basic understanding of ``Vulcanexus Easy Mode`` introduced in the :ref:`easy_mode` section.
+* An up-to-date (latest) Vulcanexus installation using one of the following installation methods:
+
+  * :ref:`linux_binary_installation`
+  * :ref:`linux_source_installation`
+  * :ref:`docker_installation`
+
+Preparation
+-----------
+
+Lets start by setting up the Vulcanexus environment.
+For this, there are two possible options:
+
+#.  Running the Vulcanexus Docker image (recommended).
```

Comment:
For this tutorial, I'd only give the docker option. Maybe the second option can be a just a note. 

I'd also run the docker with fixed IPs for the demo to work on every device. So setup the docker networks first and then run the containers with fixed IPs. 

```
docker network create --subnet=172.18.0.0/16 vulcanexus_net
docker run --net vulcanexus_net --ip 172.18.0.2 -it ubuntu bash
docker run --net vulcanexus_net --ip 172.18.0.3 -it ubuntu bash
```